src/crypto.rs

```diff
@@ -5,7 +5,9 @@ use ssz_rs::prelude::*;
 use std::fmt;
 use thiserror::Error;
 
-const BLS_DST: &[u8] = b"BLS_SIG_BLS12381G2_XMD:SHA-256_SSWU_RO_POP_";
+pub const BLS_DST: &[u8] = b"BLS_SIG_BLS12381G2_XMD:SHA-256_SSWU_RO_POP_";
```

Comment:
```suggestion
const BLS_DST: &[u8] = b"BLS_SIG_BLS12381G2_XMD:SHA-256_SSWU_RO_POP_";
```

why is the `BLS_DST` now `pub`?

Reply:
fixed.